### DISCUSSION TITLE
(Suggestion) Covid related topics maybe?

### DISCUSSION THREAD

**Discussion by Micatiou** [Monika-After-Story/MonikaModDev#7986] [general]
I know this isn't something that needs to be added in but I've been wondering, what if Monika knew about Covid. She doesn't need to know like anything specific but just some general information about covid. Like maybe there could be a topic or two referencing covid and she asks the player to be careful or something. I know this might be a lot to ask and it's totally unnecessary but it's a suggestion. I also don't know if anyone's already thought of this or if this has been discussed before so I'm sorry if I suggested something that has already been addressed. Either way, I love this mod and I am definitely not addicted haha 😅😅

**Comment by Booplicate** [comment]
There was a time limited topic, it's impossible to get it now.

**Reply by Micatiou** [reply]
Oh, it's a shame I missed it. Thank you for telling me though. I really appreciate it! 😊
